versions/development/SPEC.md

```diff
@@ -263,6 +264,49 @@ $float = (([0-9]+)?\.([0-9]+)|[0-9]+\.|[0-9]+)([eE][-+]?[0-9]+)?
 |`\'`|single quote|`\x22`|
 |`\"`|double quote|`\x27`|
 
+### Comments
+
+Comments are a useful way of providing helpful information such as workflow usage, requirements, copyright etc directly within the wdl file. Comments can be added anywhere within the `WDL` and are used to indicate text which should be ignored by an engine implementation. The one caveat to be aware of, is that within the `command` section, *ALL* text will be included in the underlying command and any lies prepended by `#` will not be ignored.
```

Comment:
`lies` => `lines`? 

And I think I'd use the active "will be included" rather than "will not be ignored"